topics/emacs/index.md

```diff
@@ -0,0 +1,13 @@
+---
+topic: emacs
+related: emacs-lisp, vim, spacemacs
+display_name: Emacs
+github_url: https://github.com/emacs-mirror/emacs
+logo: emacs.png
+short_description: An extensible, customizable, free/libre text editor — and more.
+url: https://www.gnu.org/software/emacs/emacs.html
+wikipedia_url: https://en.wikipedia.org/wiki/Emacs
+---
+Emacs is a famously extensible text editor written primarily in Emacs
```

Comment:
* Suggest omitting `famously` to keep it a bit more neutral

Reply:
Fixed.